images/miq-app-frontend/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM manageiq/manageiq-pods:backend-latest-gaprindashvili
+# FROM manageiq/manageiq-pods:backend-latest-gaprindashvili
+FROM jvlcek/manageiq-pods:backend-gaprindashvili
```

Comment:
@carbonin I'm not sure where this needs to go. Please advise.

Reply:
This will not change, when you make changes to the `miq-app` image, they will be incorporated into the new builds of the frontend image. You need to change this to test your changes, but won't need to commit it.